checks/evaluation/pinned_dependencies.go

```diff
@@ -127,15 +127,22 @@ func PinningDependencies(name string, c *checker.CheckRequest,
 		return checker.CreateRuntimeErrorResult(name, err)
 	}
 
+	// Npm installs.
+	npmScore, err := createReturnForIsNpmInstallPinned(pr, dl)
+	if err != nil {
+		return checker.CreateRuntimeErrorResult(name, err)
+	}
```

Comment:
Can you please include tests for this, also?

Reply:
I have done some validations and I'm pretty sure this code is unreachable.

Further explanation:
`PinningDependencies` creates `pr`. `pr` is a map of dependency types (e.g. npm install) and their classification (one of `pinned`, `notPinned`, `pinnedUndefined`). Then, for each dependency type in `pr` it calls `createReturnValues` to generate a score based on the classification. The score is given as follows:
- 10, if that dependency type is `pinned` or `pinnedUndefined`
- 0, if that dependency type is `notPinned`
- Error, as default case if does not meet any cases above

But, it will never happen that the classification will not meet any of the cases above. That's because `pr` is initialized with all dependency types with default classification as `pinnedUndefined`. Also, all changes to the classification will result in either `pinned` or `notPinned`. So, when the classification is scored in `createReturnValues`, it will always be one of those 3 values, and the default case is never met.
This could happen if new values were added to the classification, but that's unlikely.

Reply:
Sounds good. Thanks